nilearn/surface/surface.py

```diff
@@ -813,7 +813,10 @@ def load_surf_mesh(surf_mesh):
         if (surf_mesh.endswith('orig') or surf_mesh.endswith('pial') or
                 surf_mesh.endswith('white') or surf_mesh.endswith('sphere') or
                 surf_mesh.endswith('inflated')):
-            coords, faces = fs.io.read_geometry(surf_mesh)
+            coords, faces, header = fs.io.read_geometry(surf_mesh,
+                                                        read_metadata=True)
+            if 'cras' in header:
```

Comment:
Would you mind adding here a reference to the issue or PR ?

Reply:
Is it sufficient to do it like this?